### DISCUSSION TITLE
Levantamiento feature requests clientes

### DISCUSSION THREAD

**Discussion by DavidEhrenfeld** [Cotalker/documentation#595] [3. Ideas & Feature Requests]
Hola a todos,

Sería interesante crear una interacción mediante un formulario dentro de Cotalker para poder levantar nuevos feature requests que vayamos escuchando de clientes.

Hoy por ejemplo, en reunión con *** donde estamos haciendo seguimiento al lanzamiento masivo de solución ***, mencionaron que una mejora debería ser ordenar tareas por orden alfabético en lugar de # de tarea. Esto ya que cuando se creen nuevas tareas (y por ende procedimientos o notas) van a quedar desajustadas del orden alfabético pues la última nota quedará al final por default al estar ordenadas por número. Para *** es super importante que además del buscador general, el usuario pueda usar vista Kanban para buscar procedimientos dentro de cada producto (Estado) y la forma más intuitiva es ver todos los procedimientos (tareas) ordenados por orden alfabético.

Mil gracias :)

**Comment by acedward** [comment]
- Tareas en orden alfabético. En los filtros no tenemos por orden alfabético. Nunca habíamos pensado en usar las tareas como una base permanente, sino como tareas que se crean y terminan. Suena como algo que podemos hacer sin tanta dificultad.

En este momento, podemos ordenar por estas dimensiones:

![image](https://user-images.githubusercontent.com/3951895/146999722-29989dea-8d35-4c6b-ab5f-e51e1f44ffd1.png)
